dataplane/pkg/common/egress_interface_test.go

```diff
@@ -9,10 +9,18 @@ import (
 	"github.com/sirupsen/logrus"
 )
 
+func TestParseGatewayZero(t *testing.T) {
+	g := NewWithT(t)
```

Comment:
For some reason `gomega` encourages using `.`, but in general it's a bad practice. It may cause compile errors, and it's also bad for the readability, as one will hardly guess where `NewWithT` comes from.
I would advise removing the `.` and using `gomega.NewWithT`.

Reply:
I tried that, the tests break around the asserts.  i.e

```g.Expect(eth0).To(Equal(""))```

It complains about Equal, do you have any other pointers for me? (first time writing go :-))

Reply:
@pringlewood great you have started, it's pretty good one :) 
The `.` import means all the package exported identifiers will be declared in the importing source file's file block and are accessed without a qualifier, i.e. you can use all exported functions, without referencing the package.

So if you change to `gomega.Equal()`, should be fine.

Reply:
Thanks updated the tests